### ISSUE TITLE
Request Entity Too Large

### ISSUE THREAD

**Issue by blastur** [owncloud-docker/server#154]
When I try to sync large files (~1 GB) with my ownCloud server (based on owncloud-docker/server), I get Request Entity Too Large errors during sync. If I try to upload files via the web interface, I get "unknown error".

I've looked inside the container and it seems like the PHP settings are OK:

$ php -r "phpinfo();" 
...
post_max_size => 20G => 20G
upload_max_filesize => 20G => 20G
...

However, I also found that in my /var/www/owncloud/.htaccess contains this:
...
    php_value upload_max_filesize 513M
    php_value post_max_size 513M
    php_value memory_limit 512M
..

I believe that the upload maxsize is OK if I run PHP via cli (inside the container), but the .htaccess overrides when PHP is running via Apache somehow.

In short, I believe the actual maxsize is 513 MB and not 20G as the owncloud-docker/base environment variable OWNCLOUD_MAX_UPLOAD=20G tries to set.

**Comment by blastur** [comment]
Turns out this was caused by my nginx reverse proxy that sits infront of this owncloud container.

After adjusting the client_max_body_size config directive in nginx, owncloud is able to sync large files.

Closing this again as a non-issue.

**Comment by aratar79** [comment]
as you have configured for large files, I am using jwilder nginx reverse proxy and when I change the configuration of nginx I drop the container. add in default.conf 

server {                                                                                                                                                                                                                         server_name losdelosbares.ddns.net;                                                                                                                                                                                      listen 443 ssl http2 ;                                                                                                                                                                                                   access_log /var/log/nginx/access.log vhost;                                                                                                                                                                              ssl_session_timeout 5m;                                                                                                                                                                                                  ssl_session_cache shared:SSL:50m;                                                                                                                                                                                        ssl_session_tickets off;                                                                                                                                                                                                 ssl_certificate /etc/nginx/certs/losdelosbares.ddns.net.crt;                                                                                                                                                             ssl_certificate_key /etc/nginx/certs/losdelosbares.ddns.net.key;                                                                                                                                                         ssl_dhparam /etc/nginx/certs/losdelosbares.ddns.net.dhparam.pem;                                                                                                                                                         ssl_stapling on;                                                                                                                                                                                                         ssl_stapling_verify on;                                                                                                                                                                                                  ssl_trusted_certificate /etc/nginx/certs/losdelosbares.ddns.net.chain.pem;                                                                                                                                               add_header Strict-Transport-Security "max-age=31536000" always;                                                                                                                                                          include /etc/nginx/vhost.d/default;                                                                                                                                                                                      location / {                                                                                                                                                                                                                     proxy_pass http://losdelosbares.ddns.net;  
client_max_body_size 20G;                                                                                                                                                                      } 

Thanks.

**Comment by blastur** [comment]
I also use jwilder nginx reverse proxy. I did not have to override the whole default config, I just put "client_max_body_size 0;" as a per-host override (see jwilder docs)

**Comment by tboerger** [comment]
I don't understand why people are still using such a system like the one from jwilder at the time of nice servers like traefik or caddy :)

**Comment by aratar79** [comment]
True, you recommended it to me in your last comment when I asked for SSL, I will investigate. I use jwilder because when looking in inverse internet proxy in docker appeared the first I saw some videos, I found it very easy to use.

Thanks for the advice, regards.
